src/storage/CommonUtils.cpp

```diff
@@ -5,15 +5,20 @@
 
 #include "storage/CommonUtils.h"
 
+#include "storage/exec/QueryUtils.h"
+
 namespace nebula {
 namespace storage {
 
 bool CommonUtils::checkDataExpiredForTTL(const meta::SchemaProviderIf* schema,
                                          RowReader* reader,
                                          const std::string& ttlCol,
                                          int64_t ttlDuration) {
-  auto v = reader->getValueByName(ttlCol);
-  return checkDataExpiredForTTL(schema, v, ttlCol, ttlDuration);
+  auto v = QueryUtils::readValue(reader, ttlCol, schema);
+  if (!v.ok()) {
```

Comment:
need here log the bad status for debug? or the info will be lost.

Reply:
👌